src/testup/gem_helper.rb

```diff
@@ -58,7 +58,8 @@ def self.install(gem_name, version = Gem::Requirement.default)
       if local_path
         puts '> Installing from local copy...'
         puts "> #{local_path}"
-        Gem.install(local_path)
+        Gem.install(local_path, nil, {domain: :local})
```

Comment:
This `domain: local` options, how far back is that supported?

Reply:
Ruby 2.2, SU 2017.  It doesn't exist in Ruby 2.0, not sure about 2.1